packages/lambda-powertools-pattern-obfuscate/index.js

```diff
@@ -16,7 +16,7 @@ const ENV = process.env.ENVIRONMENT || process.env.STAGE
  */
 const FILTERING_MODE = Object.freeze({ 'BLACKLIST': 'BLACKLIST', 'WHITELIST': 'WHITELIST' })
 
-if (!process.env.DATADOG_PREFIX) {
+if (typeof process.env.DATADOG_PREFIX === 'undefined') {
```

Comment:
do we need `typeof` here? would `process.env.DATADOG_PREFIX === undefined` suffice?